### PR TITLE
Article: Java If Statement

### PR DESCRIPTION
**Pull request by nag786**
This is quick reference to if else and else if  logic using java language

### CODE REVIEW

**Comment by Rafase282** [comment]
I would rename the file to `Java-Conditionals.md` also as I stated on your other PR, you can add `java` to the code block for the syntax highlight.

File: if-else-java.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# if else and else if in Java
+-  **if:** if the if condition is true , if statement is evaluated
```
Comment (alayek) [inline if-else-java.md]:
```diff
- if the if condition is true , if statement is evaluated 
+ if the `if` condition is true , `if` statement is evaluated 
```

Note the difference between "if" and "`if`".

Comment (alayek) [inline if-else-java.md]: Also, it's not _evaluated_.  It's _executed_.

File: if-else-java.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# if else and else if in Java
+-  **if:** if the if condition is true , if statement is evaluated 
+- **else:** if the if condition is false, else statement is evaluated.
```
Comment (alayek) [inline if-else-java.md]: Same as [above](#r61236516)

File: if-else-java.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+# if else and else if in Java
+-  **if:** if the if condition is true , if statement is evaluated 
+- **else:** if the if condition is false, else statement is evaluated. 
+-  **else if:**  else if  is always used with if condition. If  **if condition** evaluated to false, the **else if statement** is executed.
```
Comment (alayek) [inline if-else-java.md]: Same as [above](#r61236516).

File: if-else-java.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+## Syntax
+if(condition)
```
Comment (alayek) [inline if-else-java.md]: Put this condition inside code highlighting syntax (multiline)

File: if-else-java.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+*Condition:* A condition is a Boolean expression which evaluates to true or false.
+
+*statement1:* When **if condition** evaluates true, the statement 1 is executed.
+
+*statement2:* If **if condition** evaluates to false, statement 2 is executed.
+
+---
+
+#### Example Using if else
+```
```
Comment (alayek) [inline if-else-java.md]:
```diff
- ```
+ ```java
```

File: if-else-java.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+int a=10;
+if(a>12) //This condition  is false.
+System.out.println("10 is not greater than 12"); //skips this statement
+else
+System.out.println("exit"); //This statement  is executed.
+```
+```
```
Comment (alayek) [inline if-else-java.md]: Add a [REPL](https://repl/it) snippet.

File: if-else-java.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+Output:
+exit
+```
+
+---
+
+### Using else if
+#### Syntax
+if(condition)
```
Comment (alayek) [inline if-else-java.md]: Same as [above](#r61236791)

File: if-else-java.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+ statement2;
+    
+ else
+ 
+   statement3;
+ 
+ ---
+
+#### Example Using else if:
+```
```
Comment (alayek) [inline if-else-java.md]: Same as [above](#r61236791)

File: if-else-java.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+      if( a>55 ){// This condition  is false.
+         System.out.print("Value of a is less than 55."); //skips this statement
+      }else if( a==15){// This condition  is true
+         System.out.println("Value of a is 15.");//This statement is executed. 
+      }else{
+         System.out.print("Exit");
+      }
+   }
+}
+```
```
Comment (alayek) [inline if-else-java.md]: Same as [above](#r61236856)

File: if-else-java.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+```
+Output:
+Value of a is 15.
+```
+
+source:source:[if else](http://www.tutorialspoint.com/java/if_else_statement_in_java.htm)
```
Comment (alayek) [inline if-else-java.md]: Refer to official Java documentation.

**Comment by alayek** [comment]
@nag786 thanks for the PR. I think @koustuvsinha was already working on something similar. 

@koustuvsinha what say you?

**Comment by koustuvsinha** [comment]
Since we are breaking up large articles into smaller pieces, this one is perfect. I would suggest file renaming to **Java-If-Statement.md** rather than Java Conditionals. Also make the appropriate headers

**Comment by abhisekp** [comment]
@nag786 **File rename:-**
```diff
- if-else-java.md
+ Java-if-else.md
```

**Comment by Rafase282** [comment]
We already have an article about [Java Control Flow](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Java-Control-Flow). Also there is https://github.com/FreeCodeCamp/wiki/pull/792 I would suggest you get in contact with the other person and work on improving the existing article.

Thank you.

**Pull request closed without merging**